Documents/API.md

```diff
@@ -132,30 +158,50 @@ All endpoints in the `/institutions` namespace require two headers (see "Authori
     * `POST` - Upload HMDA data to submission
 
 
-* `/institutions/<institution>/summary`
-    * `GET` - Summary for Financial Institution, including filing information
+* `/institutions/<institution>/filings/<period>/submissions/<submissionId>/edits`
+    * `GET`  - List of all edits for a given submission, grouped by edit type
 
     Example response, with HTTP code 200:
 
     ```json
     {
-      "id": "12345",
-      "name": "First Bank",
-      "filings": [
-        {
-          "period": "2017",
-          "institutionId": "12345",
-          "status": "not-started"
-        },
-        {
-          "period": "2016",
-          "institutionId": "12345",
-          "status": "completed"
-        }
-      ]
+      "syntactical": {
```

Comment:
Following up on this: With the updates you made, the output now looks like this:
```
{
  "syntactical": {
    "edits": [
      {
        "edit": "S020",
        "lars": [
          {
            "lar": { "loanId": "58374834663421" }
          }
        ]
      },
      {
        "edit": "S010",
        "lars": [
          {
            "lar": { "loanId": "8299422144889800" }
          }
        ]
      }
    ]
  },
  "validity": {
    "edits": [
      {
        "edit": "V480",
        "lars": [
          {
            "lar": { "loanId": "45298499442" }
          }
        ]
      },
      {
        "edit": "V220",
        "lars": [
          {
            "lar": { "loanId": "469686954747885" }
          }
        ]
      }
    ]
  },
  "quality": {
    "edits": []
  },
  "macro": {
    "edits": []
  }
}
```